### DISCUSSION TITLE
How to get th real annotations data.

### DISCUSSION THREAD

**Discussion by MedAliMimouni** [mozman/ezdxf#723] [Q&A]
Is it possible to get the geo data, like in geopandas?

I mean the start point, the end point, and the middle points for a linestring.

if you have some helpful links please throw them hear.

Thanks

**Comment by mozman** [comment]
I am not sure what you are looking for, but the [LWPOLYLINE](https://ezdxf.mozman.at/docs/dxfentities/lwpolyline.html) entity is very similar to a line-string, except it can contain arcs. You can get the vertices by the method `LWPolyline.vertices_in_wcs()`.

**Comment by MedAliMimouni** [comment]
@mozman  Thanks for the response, I'm trying to build a labled dataset and I want to read every shape.

Should I read every entity?
